scout/server/blueprints/institutes/controllers.py

```diff
@@ -131,6 +132,45 @@ def institute(store, institute_id):
     return data
 
 
+def populate_beacon_form(institute_obj):
+    """Populate the form select of scout.server.blueprints.institutes.forms.BeaconDatasetForm with data.
+    Available data is a controlled dictionary of dataset names depending of the datasets already existing in the Beacon server
+
+    Args:
+        institute_obj(dict) An institute object
+    """
+    beacon_form = BeaconDatasetForm()
+
+    if current_user.is_admin is False:
+        LOG.warning("User must be an admin to create a beacon dataset")
+        return beacon_form
+
+    if not all(
+        [current_app.config.get("BEACON_URL") and current_app.config.get("BEACON_TOKEN")]
+    ):  # Send a request to the beacon to collect the existing datasets
+        LOG.warning(
+            "Both BEACON_URL and BEACON_TOKEN params are required in Scout configuration file to create a new Beacon dataset"
```

Comment:
A bit same here; maybe warn if one is set but not the other?

Reply:
This warning would never show if the params in the config file are set correctly. But it's useful to see a warning if there is a misconfiguration, so one fixes it. I'm going to remove the warning in the code above, but I'd like to leave this one as it is.

Reply:
Sure; we should never see it in prod anyway. Consider my suggestion, but its fine.

Reply:
MM I don't think the code suggestion works as intended, If I remove both params from the conf file it executes the lines below the if anyway. I'll fix!